wordpress-mobile/WordPress-iOS
# [Dark Mode ]NUX Button Loading

Fixes [#140](https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/issues/140)
Refs. [#142](https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/142)

This PR update the WordPress Authenticator pod. The new version includes a fix where the NUX button activity indicator is visible in light/dark mode

![dark-mode_nux-button-loading](https://user-images.githubusercontent.com/912252/66059707-61009900-e534-11e9-94cb-497936f55ed9.jpg)

## To test:
- Run this branch on iOS 13
- If you are logged in sign out
- From the login, use the WordPress login and add your email. Then press _Next_.
- You should see the loader
- run it on iOS 12 to see everything works fine

**Podfile will be updated as soon [#142](https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/142) will be merged**

Update release notes:
